apps/server/src/modules/authorization/authorization.service.ts

```diff
@@ -61,6 +63,27 @@ export class AuthorizationService extends BasePermissionManager {
 		}
 	}
 
+	async hasPermissionsByReferences(
```

Comment:
this should be discussed with people that have worked on the authorisation service. I dont know if its intended to handle multiple permissions somewhere else

Reply:
ideally cedric, sergej, or uwe